packages/picasso/src/InputLimitAdornment/InputLimitAdornment.tsx

```diff
@@ -22,6 +22,7 @@ const useStyles = makeStyles<Theme>(styles, {
 })
 
 const hasRemainingCounter = (counter: CounterType) => counter === 'remaining'
+const characters = (count: number) => (count === 1 ? 'character' : 'characters')
```

Comment:
nit: I know it is really simple but not sure about the function name 🤔 
```suggestion
const formatCharacters = (count: number) => (count === 1 ? 'character' : 'characters')
```

Reply:
I didn't like the name either :smiley: